examples/resources/powermax_snapshot/provider.tf

```diff
@@ -0,0 +1,25 @@
+terraform {
+  required_providers {
+    powermax = {
+      source = "dell/powermax"
+    }
+  }
+}
+
+# provider "powermax" {
+#   username      = var.username
+#   password      = var.password
+#   endpoint      = var.endpoint
+#   serial_number = var.serial_number
+#   pmax_version  = var.pmax_version
+#   insecure      = true
+# }
+
+provider "powermax" {
+  username      = "smc"
+  password      = "smc"
+  endpoint      = "https://10.225.104.37:8443"
```

Comment:
can you remove actual data?

Reply:
Yes sorry thought I cleaned this up after testing, will do!